### PR TITLE
Allow to reload extension from within error modal

### PR DESCRIPTION
**Pull request by buberdds**
Fixes https://github.com/oasisprotocol/oasis-wallet-web/issues/1853

Don is fine with ext layout variant.

### CODE REVIEW

File: src/app/components/Button/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+})
+
+export function BrandButton(props: BrandButtonProps) {
+  const { brandVariant = 'blue', icon, size = 'large', ...rest } = props
+  const theme = useContext(ThemeContext)
+  const { backgroundColor, color } = getBrandVariantColors(theme)[brandVariant]
+  const styledIcon = useMemo(() => {
+    if (icon) {
+      return cloneElement(icon, {
+        size: '22px',
```
Comment (lukaw3d) [inline src/app/components/Button/index.tsx:40]: This doesn't go with customizable props.size. Maybe 1.6em

File: src/app/components/Button/index.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      icon={styledIcon}
+      style={{
+        backgroundColor,
+        color,
+        borderColor: backgroundColor,
+      }}
+      size={size}
+    />
+  )
+}
```
Comment (lukaw3d) [inline src/app/components/Button/index.tsx:58]: Other than icon size this is the same as just:
```tsx
interface BrandButtonProps extends ButtonExtendedProps {
  brandVariant?: 'brand-blue' | 'brand-gray-medium'
}
export function BrandButton(props: BrandButtonProps) {
  const { brandVariant = 'brand-blue', icon, size = 'large', ...rest } = props
  return <StyledButton {...rest} icon={icon} primary color={brandVariant} size={size} />
}
```

**Review by lukaw3d — APPROVED** [review]

File: src/app/components/FatalErrorHandler/index.tsx [hunk 3]
```diff
@@ -82,40 +76,38 @@ export function FatalErrorHandler({ children }: Props) {
...
-            style={{
-              backgroundColor: normalizeColor('brand-gray-medium', theme),
-              color: normalizeColor('brand-gray-extra-dark', theme),
-              borderColor: normalizeColor('brand-gray-medium', theme),
-            }}
-            size="large"
           />
         </Box>
+        {runtimeIs === 'extension' && (
+          <Box align={isMobile ? 'stretch' : 'end'} margin={{ bottom: isMobile ? 'xlarge' : 'large' }}>
```
Comment (lukaw3d) [inline src/app/components/FatalErrorHandler/index.tsx:103]: preview
image

Comment (buberdds) [reply src/app/components/FatalErrorHandler/index.tsx:103]: will ask Don tomorrow if we want to move this button up

Comment (buberdds) [reply src/app/components/FatalErrorHandler/index.tsx:108]: Screenshot from 2024-04-02 11-28-37

**Pull request merged**